docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+```
+
+### parseMarkdown
+
+Parse selected paragraph text to edit (in draft mode) to HTML.
+
+```scala
+parseMarkdown(
+  str: String
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `str` | `String` | **Required.** The text of the selected paragraph to edit. |
+
+Example
+
+```js
+const currentParagraph = 'This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:';
+Openlaw.parseMarkdown(currentParagraph);
+```
+
+**Response**
+
+Returns selected paragraph text parsed to HTML.
+
+Example
+
+```html
+<p class='no-section'>This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:</p>
+```
+
+### renderParagraphForEdit
+
+Render paragraph of agreement for editing.
+
+```scala
+renderParagraphForEdit(
+  agreement: StructuredAgreement,
+  index: Int
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `agreement` | [`StructuredAgreement`](#structuredagreement) | **Required.** A [`StructuredAgreement` object](#structuredagreement) representing the agreement to be rendered. |
+| `index` | `Int` | **Required.** The index number of the paragraph to be rendered for edit. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const agreements = Openlaw.getAgreements(executionResult.executionResult);
+Openlaw.renderParagraphForEdit(agreements[0].agreement, 1);
+```
+
+**Response**
+
+Returns the HTML contents of the paragraph to be rendered for edit as a string.
+
+Example
+
+```html
+<p class='no-section'>This Advisor Agreement is entered into between [[Company Name]] ("Corporation") and John Smith ("Advisor") as of September 19, 2018 ("Effective Date"). The parties agree as follows:</p>
+```
+
+### getTypes
+
+Get all [markup language](/markup-language/) variable types for use in markdown editor.
+
+```scala
+getTypes: js.Array[String]
+```
+
+**Parameters**
+
+None
+
+**Response**
+
+Returns array of [markup language](/markup-language/) variable types as strings.
+
+Example
+
+```js
+[
+  "Collection",
+  "Address",
+  "Choice",
+  "Date",
+  "DateTime",
+  "EthAddress",
+  "EthereumCall",
+  "StripeCall",
+  "Identity",
+  "Number",
+  "Period",
+  "SmartContractMetadata",
+  "Structure",
+  "Template",
+  "Text",
+  "Validation",
+  "YesNo"
```

Comment:
Is there a way we could link to these types' descriptions in the markup language docs? Or is that unnecessary?

Reply:
Probably isn't necessary to link for each if we just provide a link to the markup language docs in explaining what the response is right before this list. I changed to this:

Returns array of input and specialized variable types as strings. Learn more about these different types in our [Markup Language documentation](/markup-language/).

Reply:
perfect!